ksqldb-rest-app/src/test/java/io/confluent/ksql/rest/server/execution/ListTopicsExecutorTest.java

```diff
@@ -59,6 +63,7 @@ public void setUp() {
           engine.getServiceContext().getSchemaRegistryClientFactory(),
           engine.getServiceContext().getConnectClient()
     );
+    customExecutors = new CustomExecutors(new DefaultConnectServerErrors());
```

Comment:
nit: most of the updated tests introduce
```
private static final CustomExecutors CUSTOM_EXECUTORS
```
instead, which I think is cleaner. Can we keep these test updates consistent?

Reply:
Done in https://github.com/confluentinc/ksql/pull/8480/commits/7ac1fe2c1b271aec0498c7f1315b34f0681bb7a4#diff-4a4e423d225fa849b4fab3a3e83448875660ab295021d824c67562715b987b0a.